assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.js

```diff
@@ -88,6 +88,12 @@ function AudienceSegmentationSetupCTAWidget( { Widget, WidgetNull } ) {
 		)
 	);
 
+	const dismissCountLoaded = useSelect( ( select ) =>
+		select( CORE_USER ).hasFinishedResolution( 'getPromptDismissCount', [
+			AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION,
+		] )
+	);
+
```

Comment:
`getPromptDismissCount` doesn't have a resolver, so it doesn't make sense to try to wait for it to resolve - `dismissCountLoaded` will only ever be `false`. We can instead check the `getDismissedPrompts` resolver.

```suggestion
	const dismissedPromptsLoaded = useSelect( ( select ) =>
		select( CORE_USER ).hasFinishedResolution( 'getDismissedPrompts', [] )
	);
```